samples/frameworks/react/ts/README.md

```diff
@@ -0,0 +1,65 @@
+---
+page_type: sample
+languages:
+  - typescript
+products:
+  - azure
+  - azure-event-hubs
+  - azure-service-bus
+  - azure-storage
+urlFragment: react-typescript
+---
+
+# Azure SDK samples for React (TypeScript)
+
+This sample application shows how to use the TypeScript client libraries for Azure in some common scenarios.
+
+In this sample, we build a simple Todo application in React using [create-react-app][react] and integrating with various Azure services
+
+- Integration with Azure EventHubs showcases a mechanism for multiple instances to consume EventHub messages produced by any one of them.
+- Integration with Azure ServiceBus showcases a mechanism for any single instance to consume ServiceBus messages. When multiple windows are open, only one of them will receive any given message.
+- Integration with Azure Storage Blob showcases a mechanism for fetching Azure blobs and displaying them in the browser.
+
+## Prerequisites
```

Comment:
Documentation is hard! I'd love to hear about where I can improve the docs. Especially interested in how to make the README better. There's a non-trivial amount of setup and permission management that I did not know how to properly capture.